crates/core/tedge_mapper/src/c8y_converter.rs

```diff
@@ -114,6 +117,9 @@ impl Converter for CumulocityConverter {
             self.try_convert_measurement(input)
         } else if input.topic.name.starts_with("tedge/alarms") {
             self.try_convert_alarm(input)
+        } else if input.topic.name.starts_with("tedge-internal/alarms") {
+            // Do not try to convert as this is an internal message
+            Ok(vec![])
```

Comment:
This case can be removed. If the reconciliation loop is rewritten to use its own MQTT connection, then the main loop can avoid to subscribe to the `tedge-internal/alarms` topic.